modules/cnf-running-the-performance-creator-profile-offline.adoc

```diff
@@ -0,0 +1,228 @@
+// Module included in the following assemblies:
+// Epic CNF-792 (4.8)
+// * scalability_and_performance/cnf-create-performance-profiles.adoc
+
+[id="running-the-performance-creator-profile-wrapper_{context}"]
+= Running the Performance Profile wrapper script
+
+The performance profile wrapper script simplifies the running of the Performance Profile Creator (PPC) tool. It hides the complexities associated with running `podman` and specifying the mapping directories
+and it enables the creation of the performance profile.
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware.
+* A node with `podman` and OpenShift CLI (`oc`) installed.
+* Access to `must-gather` data.
+
+[NOTE]
+====
+The user is expected to run `must-gather` and compress the output. For more information see the section Gathering data about your cluster under Support
+====
+
+.Procedure
+
+. Create a file on your local machine named for example `run-perf-profile-creator.sh`:
```

Comment:
Create a file on your local machine named, for example, `run-perf-profile-creator.sh`: